### ISSUE TITLE
[nist-branch] Ensure all license information is clear

### ISSUE THREAD

**Issue by dstebila** [open-quantum-safe/liboqs#249]

**Comment by dstebila** [comment]
Also need LICENSE.txt file for Kyber. @tlepoint can you add this?

**Comment by dstebila** [comment]
NewHopeNIST is licensed under public domain; need to add a LICENSE.txt file to this effect.

**Comment by dstebila** [comment]
Also as part of this issue we should add a Travis test that checks to make sure every upstream folder contains a LICENSE.txt file.

**Comment by dstebila** [comment]
NewHope added by commit 218a395698278ebb7a0128b757ac545bd4de2f89

**Comment by dstebila** [comment]
Here is something we could use as `.travis/license-check.sh`:

```bash
#!/bin/bash

###
# Checks that all upstream implementations include a LICENSE/LICENSE.txt file.
###

PRINT_GREEN="tput setaf 2"
PRINT_RED="tput setaf 1"
PRINT_RESET="tput sgr 0"

RET=0

for d in $(find src/kem -name upstream); do
    if [[ ! -f ${d}/LICENSE ]]; then
        if [[ ! -f ${d}/LICENSE.txt ]]; then
            ${PRINT_RED}
            echo "No LICENSE/LICENSE.txt file in ${d}."
            RET=1
            ${PRINT_RESET}
        fi
    fi
done

if [[ ${RET} == 0 ]]; then
    ${PRINT_GREEN}
    echo "All LICENSE/LICENSE.txt files present.";
    ${PRINT_RESET}
fi

exit ${RET}
```

**Comment by dstebila** [comment]
Fixed by #267
